benchmarks.md

```diff
@@ -0,0 +1,16 @@
+## Chunkgen benchmarks
```

Comment:
**WARNING: THE FOLLOWING BENCHMARKS SHOULD NOT BE USED FOR ANY DEBATE, THEY'RE ONLY HERE FOR REFERENCE PURPOSES. DIFFERENT HARDWARE MIGHT AND WILL GIVE DIFFERENT RESULTS.**